libs/documentation/src/lib/storybook/autocomplete/autocomplete-checkbox/autocomplete-checkbox.component.ts

```diff
@@ -0,0 +1,48 @@
+import { Component, OnInit } from '@angular/core';
+import { SDSAutocompletelConfiguration, SDSSelectedItemModel, SelectionMode } from '@gsa-sam/components';
+import { SampleAutocompleteData } from '../services/autocomplete-sample.data';
+import { AutocompleteSampleDataService } from '../services/autocomplete-sample.service';
+
+@Component({
+  templateUrl: './autocomplete-checkbox.component.html',
+  selector: `sds-autocomplete-checkbox-demo`,
+  providers: [AutocompleteSampleDataService],
+})
+export class AutocompleteCheckbox implements OnInit {
+  private data = SampleAutocompleteData;
```

Comment:
var is not used anywhere

Reply:
all other components have this, i just copied it.

Reply:
okay..? just because a mistake is in other demo doesn't mean you should propagate it. If it's useless there is no need to keep it

Reply:
removed